packages/client/src/index.ts

```diff
@@ -39,6 +39,20 @@
  * await libp2p.peerRouting.findPeer(peerIdFromString('QmFoo'))
  * ```
  *
+ * ### Caching
+ *
+ * By default, the client caches responses in browser environments for a duration of 5 minutes.
```

Comment:
I would change this to more accurately state that we set the `x-cache-expires` header on the response which only works in environments that respsect it (i.e. browsers)

Reply:
`x-cache-expires` is just an arbitrary name. The browser doesn't inspect it.